examples/worlds/lift_drag.sdf

```diff
@@ -226,7 +231,7 @@
           </limit>
           <xyz>0.0 0.0 -1.0</xyz>
           <dynamics>
-            <damping>0.0001</damping>
+            <damping>0.001</damping>
```

Comment:
Curious why this was changed.

Reply:
I think I was either looking to reduce joint noise or it changed as part of the PID tune when using the joint velocity controller (I changed the example from using the apply force plugin as I wanted a constant joint velocity for the lift-drag forces).  Pretty sure it can be reverted if needed.

Reply:
Hmm, if we don't use force commands then we would not need to do this. I'm not sure if this will effect physics calculations. However, it is possible to directly write angular velocities to the joint if `<use_force_commands>` is not set.

To be honest this does not really matter.